tests/kernels/test_celu.cpp

```diff
@@ -49,6 +49,51 @@ class CeluTest : public KernelTest,
 
     void TearDown() override {}
 
+    virtual void init_tensor(runtime::runtime_tensor &tensor) override {
+        auto dtype = tensor.datatype();
+        switch (dtype) {
+        case dt_float16: {
+            std::random_device rd;
+            std::mt19937 gen(rd());
+            std::uniform_real_distribution<float> dis(0.0f, 1.0f);
```

Comment:
should generate negative numbers also.

Reply:
Probabilistic errors cuz by negative numbers

Reply:
done